ecole-doctorale-de-sciences-po-author-date-french.csl

```diff
@@ -0,0 +1,440 @@
+            <text variable="edition" text-case="capitalize-first" suffix="."/>
+          </else>
+        </choose>
+      </if>
+      <else-if type="article-journal article-magazine" match="any">
+        <group font-style="normal">
+          <choose>
+            <if variable="issued">
+              <date variable="issued">
+                <date-part name="day" suffix=" "/>
+                <date-part name="month" suffix=" "/>
+                <date-part name="year"/>
+              </date>
+              <text macro="volume" prefix=", "/>
+            </if>
+            <else>
+              <text macro="volume" text-case="capitalize-first"/>
+            </else>
+          </choose>
+        </group>
+      </else-if>
+    </choose>
+    <text macro="issue" prefix=" "/>
+  </macro>
+  <macro name="volume">
+    <choose>
+      <if is-numeric="volume">
+        <text term="volume" form="short" suffix=".&#160;"/>
+        <text variable="volume"/>
+      </if>
+      <else>
+        <text variable="volume"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issue">
+    <choose>
+      <if is-numeric="issue">
+        <text term="issue" form="short" suffix="&#160;"/>
+        <text variable="issue"/>
+      </if>
+      <else>
+        <text variable="issue"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <choose>
+      <if is-numeric="collection-number">
+          <text variable="collection-title" prefix="(" suffix=""/>
+        <text variable="collection-number" prefix=" ; n˚&#160;" suffix=")"/>
+      </if>
+      <else>
+          <text variable="collection-title" prefix="(" suffix=")"/>
+      </else>
+    </choose>
+  </macro>
+<citation et-al-min="2" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" givenname-disambiguation-rule="primary-name">
+    <layout prefix="&#160;(" suffix=")" delimiter="; ">
+      <choose>
+        <if position="ibid-with-locator">
+          <group delimiter=", ">
+            <text term="ibid" text-case="capitalize-first" font-style="italic" suffix="."/>
+            <text variable="locator" prefix="p. "/>
+          </group>
+        </if>
+        <else-if position="ibid">
+          <text term="ibid" text-case="capitalize-first" font-style="italic"/>
+        </else-if>
+        <else>
+          <group delimiter=" ">
+            <text macro="contributors-note" suffix=","/>
+            <text macro="year"/>
+          </group>
+          <group prefix=", ">
+            <label variable="locator" form="short" suffix="&#160;"/>
+            <text variable="locator"/>
+          </group>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography et-al-min="4" et-al-use-first="3">
+    <sort>
+      <key macro="author" names-min="3" names-use-first="3" />
+      <key variable="issued" sort="descending"/>
+    </sort>
+    <layout>
+        <text macro="author" suffix=". "/>
```

Comment:
more elegant to handle all of these suffixes with a single `delimiter=". "`

Reply:
i don't understand how to do that

Reply:
could you help me, please ?